api/RTCPeerConnectionIceErrorEvent.json

```
@@ -185,6 +234,54 @@
               "version_added": "77"
             }
           },
+          "status": {
+            "experimental": false,
```

Unfortunately, it's too soon to flip this to false https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status

```suggestion
            "experimental": true,
```

---

@teoli2003 did you mean to resolve this?

---

I don't really know why I resolve without fixing. I may have mixed two comments. Anyway, committed now!